config/config-sil-ocpp.yaml

```diff
@@ -161,9 +161,13 @@ active_modules:
       display_message:
         - module_id: display_message
           implementation_id: display_message
-      connector_zero_sink:
+      evse_manager_energy_sink:
         - module_id: grid_connection_point
           implementation_id: external_limits
+        - module_id: evse_manager_1_sink
+          implementation_id: external_limits
+        - module_id: evse_manager_2_sink
+          implementation_id: external_limits
```

Comment:
Might be a bit problematic here, as this ordering needs to comply with the ordering of the EVSE managers above

Reply:
This can be addressed with https://github.com/EVerest/everest-framework/pull/206